templates/publisher/measure.html

```diff
@@ -0,0 +1,102 @@
+{% extends "_layout.html" %}
+
+{% block title %}
+  Publisher
+{% endblock %}
+
+{% block content %}
+  <div id="main-content" class="u-no-margin--top">
+    <section class="p-strip is-shallow">
+      <div class="row">
+        <p><a href="/snaps">My snaps&nbsp;&rsaquo;</a></p>
+        <h1 class="u-no-margin--top">{{ snap_title }}</h1>
+      </div>
+    </section>
+    <section class="p-strip is-shallow u-no-padding--top u-no-padding--bottom">
+      <nav class="p-tabs">
+        <ul class="p-tabs__list" role="tablist">
+          <li class="p-tabs__item" role="presentation">
+            <a href="/snaps/{{ package_name }}/measure" class="p-tabs__link" tabindex="0" role="tab" aria-controls="section1" aria-selected="true">Measure</a>
+          </li>
+        </ul>
+      </nav>
+    </section>
+    <section class="p-strip is-shallow">
+      <div class="row">
+        <div class="grid">
+          <div class="col-3">
+            <select class="p-form__control" disabled="disabled">
+              <option value="30d"{% if metric_range == '30d' %} selected="selected"{% endif %}>30 days</option>
+            </select>
+          </div>
+          <div class="col-3">
+            <select class="p-form__control" disabled="disabled">
+              <option value="all">All tracks and branches</option>
+            </select>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="u-clearfix">
+          <h3 class="u-float--left">Installs</h3>
+          <strong class="p-heading--three u-float--right u-no-margin--top">{{ installs_total }}</strong>
+        </div>
+        <div id="installs_metrics" class="snapcraft-metrics__graph">
+        </div>
+      </div>
+    </section>
+    <!-- <div class="row">
```

Comment:
Maybe hide it with flask template comment? `{# #}`

Reply:
Done! Thanks @bartaz